README.md

```diff
@@ -13,8 +20,9 @@ This repository contains the scripts and Terraform configurations required to se
 
 A pipeline should be created for each Bosh release this repository is currently building releases for. The `build-dev-release` jobs should trigger when pull requests are raised against the Bosh release's repository. The `build-final-release` job should trigger when new commits are added to the branch used to build final releases.
 
+## Overriding variables
+
 You can override some variables to customise the deployment:
  * `BRANCH` current branch to pull and use. e.g. `BRANCH=$(git rev-parse --abbrev-ref HEAD)`
  * `CONCOURSE_URL`, `CONCOURSE_ATC_PASSWORD`: to point to a different concourse with the given credentials.
- * `STATE_BUCKET_NAME`, `RELEASES_BUCKET_NAME`: use alternative state and releases buckets.
-
+ * `STATE_BUCKET_NAME`, `RELEASES_BUCKET_NAME`: use alternative state and releases buckets. Note that bucket for releases gets created in the `setup` pipeline. You have to thus chose a bucket name, that policy of the concourse you use allows creating. See our [policies repository](https://github.gds/government-paas/aws-account-wide-terraform/blob/master/policies-json/concourse_manage_s3_buckets.json) for details.
```

Comment:
Ahh. I think it'd be sufficient to say "Concourse must have an IAM policy that allows it to create and manage the buckets specified" without linking to our policies (which people outside our team can't see at the moment anyway).